platformio-example.ini

```diff
@@ -16,6 +16,7 @@ src_dir = multigeiger
 
 [env:geiger]
 board = heltec_wireless_stick
+board_build.partitions = min_spiffs.csv
```

Comment:
does this work for all currently supported devices?

is the result the same as with the ardiuno settings described in the docs?

Reply:
It works with the old wifi Kit 32 which have 4 MB memory.